Avoid link-time error

../src/.libs/libgcrypt.so: undefined reference to `__udiv_qrnnd'

by choosing v8-like insns for 32-bit v9 mode too.

---
 mpi/longlong.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: libgcrypt-1.7.2/mpi/longlong.h
===================================================================
--- libgcrypt-1.7.2.orig/mpi/longlong.h
+++ libgcrypt-1.7.2/mpi/longlong.h
@@ -1293,7 +1293,7 @@ typedef unsigned int UTItype __attribute
 	     "rJ" ((USItype)(al)),                                      \
 	     "rI" ((USItype)(bl))                                       \
 	   __CLOBBER_CC)
-# if defined (__sparc_v8__) || defined(__sparcv8)
+# if defined (__sparc_v8__) || defined(__sparcv8) || defined(__space_v9__)
 /* Don't match immediate range because, 1) it is not often useful,
    2) the 'I' flag thinks of the range as a 13 bit signed interval,
    while we want to match a 13 bit interval, sign extended to 32 bits,
